static/app/bootstrap/initializeSdk.tsx

```diff
@@ -209,3 +210,7 @@ export function isFilteredRequestErrorEvent(event: Event): boolean {
 
   return false;
 }
+
+export function isEventWithFileUrl(event: Event): boolean {
+  return !!event.request?.url?.startsWith('file://');
```

Comment:
should we get rid of the denyDomains or whatever that option was that didn't work

Reply:
Yup - see the last commit there.